share/OpenMS/CHEMISTRY/Enzymes.xml

```diff
@@ -318,9 +328,14 @@
       <ITEM name="CTermGain" value="OH" type="string" />
       <ITEM name="PSIID" value="MS:1001956" type="string" />
       <ITEM name="XTandemID" value="[X]|[X]" type="string" />
+      <ITEM name="CruxID" value="no-enzyme" type="string" />
```

Comment:
hmm, do you think this is what `no-enzyme ([X]|[X])` means? I thought that would be `([]|[])` if it cuts everywhere but we can ask